### PR TITLE
feat(core): Add onStop callback to SpotLightTourProvider props

### PR DESCRIPTION
**Pull request by alejo0o**
This PR adds the possibility to run some events or actions after the tour stops by adding a new prop to the SpotLightTourProvider called onStop.
It solves issue #89

### CODE REVIEW

**Review by JoseLion — APPROVED** [review]
Nice job @alejo0o! I left a minor suggestion, but I think this one's ready to merge. Thanks! 🎉 

PS: Nice testing! Clear and concise 🙂

File: package/src/lib/SpotlightTour.provider.tsx [hunk 1]
```diff
@@ -52,6 +53,14 @@ export interface SpotlightTourProviderProps {
    * **NOTE:** You can also override this behavior on each step configuration.
    */
   onBackdropPress?: BackdropPressBehavior;
+  /**
+   * Handler which gets executed when {@link SpotlightTour.stop|stop} is
+   * invoked. It receives the {@link OnStopBehavior} so
+   * you can access the `current` step index where the tour stopped
+   * and a bool value `isLast` indicating if the step where the tour stopped is
+   * the last one.
+   */
```
[the remark this answers is not in the document]
Comment (alejo0o) [reply package/src/lib/SpotlightTour.provider.tsx:62]: 🙇

File: package/src/lib/SpotlightTour.context.ts [hunk 2]
```diff
@@ -86,6 +86,17 @@ export type BackdropPressBehavior =
   | "stop"
   | ((options: SpotlightTour) => void);
 
+export interface OnStopBehavior {
```
[the remark this answers is not in the document]
Comment (alejo0o) [reply package/src/lib/SpotlightTour.context.ts:89]: Sure 👍

**Comment by JoseLion** [comment]
@all-contributors please add @alejo0o for code

**Pull request merged**